src/Paysera/WalletApi/Entity/Inquiry/InquiryItem.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+class Paysera_WalletApi_Entity_Inquiry_InquiryItem
+{
```

Comment:
Maybe we should provide available types for items?

By the way, title will not be used yet with current available type.